### PR TITLE
Further Fixes and Tweaks for 1.7

### LINKED ISSUE #367: Nomi CEu HM Changes [Add to this with HM QB Issues!]
### Nomi CEu Version

1.6.1-beta-2

### Related Problem

I'm intending to open this issue to remark observations on Hard/Expert mode playthrough that can be seen as oversights or just not enough elaboration on the difference from Normal mode.

### Your Solution

Elaborate over the observations and seek for possible fixes if necessary for them. Each comment will present a different observation with most likely a possible fix for it.

### How Will Your Solution Work

----------

### Alternatives You've Considered

----------

### Additional Information

_No response_

### LINKED ISSUE #371: Normal mode Quest 504 "Brewery" should also accept HV+ breweries
### Nomi CEu Version

1.6.1-beta-2

### Related Problem

_No response_

### Your Solution

I used the pyrolysis oven to make biomass and wood tar so I did not build a brewery at LV or MV levels - I made one at HV for lubricant. But that one is not recognized by the quest.

### How Will Your Solution Work

As the LV/MV brewery are non essential to progressing - the quest should also accept higher tier breweries.

### Alternatives You've Considered

Craft a LV/MV brewery and immediately recycle it after completing the quest

### Additional Information

_No response_

### LINKED ISSUE #417: Fluid Rig in the QB
### Nomi CEu Version

1.6.1a

### Related Problem

The fluid rig quest is kinda misleading for a lot of beginners, wording implies that all veins are finite when that's not the case

### Your Solution

Rewrite some of the confusing text in the quest and make it clear that void oil vein is infinite and for other veins player might see the depleted rate in JEI

### How Will Your Solution Work

It will make it more clear that players can just place a machine and not have to move it ever again

### Alternatives You've Considered

-

### Additional Information

_No response_

### LINKED ISSUE #442: Add Genesis quest to create storage crates
### Nomi CEu Version

1.6.1a

### Related Problem

_No response_

### Your Solution

There is a genesis quest to make a drawer, but the player is left on their own to discover storage crates. Those should be exposed around the same time.

### How Will Your Solution Work

Add a genesis tier quest for the player to make a small storage crate.

### Alternatives You've Considered

Adding storage crate to the drawer quest.

### Additional Information

_No response_

### LINKED ISSUE #449: Titanium Tetrachloride quests requires a bucket
### Nomi CEu Version

1.6.1a

### Launcher

Minecraft Lauchner

### Configurations or Mods Changed

Disable seasonal events in EnderIO
Reduced ladder climbing speed from 4 to 3

### Environment

Singleplayer / Peaceful

### Mode

Normal Mode

### What Happened

The Titanium Tetrachloride quests requires that the fluid is stored in a bucket while all other fluid quests also accept EnderIO tanks (which is my preferred method for manual chemistry) 

### Expected Behavior

All fluid containers should be allowed for this quest.

### Reproduction Steps

1. Have the Titanium Tetrachloride in an EnderIO tank and try to complete the quest
2. The quest does not detect the liquid
3. Transfer the liquid into a vanilla bucket
4. The quest can be completed

### Logs

_No response_

### Additional Information

_No response_

### LINKED ISSUE #453: Macerate Charcoal Blocks
### Nomi CEu Version

1.6.1a

### Related Problem

_No response_

### Your Solution

Allow macerators to turn one Charcoal Block into nine Charcoal Dust using an analogous process to the Coal Block.

### How Will Your Solution Work

Since Charcoal Blocks are from the Chisel mod and not GregTech, perhaps a small addon could add this recipe to Macerators (preferably with the same stats and voltage tier as with the Coal Block). (Or use a previous tweaking mod already present in Nomi-CEu.)

### Alternatives You've Considered

Just craft the block into 9 charcoal and macerate them individually.

### Additional Information

Here are the stats for the macerator recipe for the Coal Block:

Total Cost: 216 EU
Usage: 2 EU/t (ULV)
Duration: 5.40 s

### PR DESCRIPTION
**Pull request by tracer4b**
This PR contains further tweaks (mainly QB related) for 1.7

- [x] Closes #367 
- [x] Closes #371 
- [x] Closes #417 
- [x] Closes #442  
- [x] Closes #449
- [x] Closes #453
- [x] Add a quest for Multiblock Builder
- [x] Fix some Assembly Line recipe changes from #451
- [x] Add GT machine recipes for some crafting components to ease bulk crafting times

### CODE REVIEW

**Comment by IntegerLimit** [comment]
Maybe https://github.com/Nomi-CEu/Nomi-CEu/issues/453 can fit here too.

**Review by IntegerLimit — APPROVED** [review]

File: overrides/scripts/_oreDict.zs [hunk 1]
```diff
@@ -2604,6 +2611,13 @@ recipes.addShapeless(<metaitem:ingotRedAlloy>, [<enderio:item_alloy_ingot:3>]);
 <ore:questbookParallelControlHatch>.add(<metaitem:gcym:parallel_hatch.zpm>);
 <ore:questbookParallelControlHatch>.add(<metaitem:gcym:parallel_hatch.uv>);
 
+<ore:questbookSteamBoiler>.add(<gregtech:machine:1>);
+<ore:questbookSteamBoiler>.add(<gregtech:machine:2>);
+<ore:questbookSteamBoiler>.add(<gregtech:machine:3>);
+<ore:questbookSteamBoiler>.add(<gregtech:machine:4>);
+<ore:questbookSteamBoiler>.add(<gregtech:machine:5>);
+<ore:questbookSteamBoiler>.add(<gregtech:machine:6>);
+
```
Comment (IntegerLimit) [inline overrides/scripts/_oreDict.zs:2620]: Unused OreDict?

Comment (IntegerLimit) [inline overrides/scripts/_oreDict.zs:2620]: Unused OreDict?

**Review by IntegerLimit — CHANGES_REQUESTED** [review]
Should some stuff in [`1956a8d` (#470)](https://github.com/Nomi-CEu/Nomi-CEu/pull/470/commits/1956a8dbcae241c4bf9d7627781b57a15c97913f) apply to HM too?

File: overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json [hunk 2]
```diff
@@ -1648,7 +1648,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Your first RF power generator!\n\nKeep in mind that §eDynamos only output power when burning through fuel, and only the block attached directly to the coil will get provided with power§f.",
```
Comment (IntegerLimit) [inline overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json:1651]: Reverse this change

File: overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json [hunk 3]
```diff
@@ -22230,13 +22220,16 @@
...
+              "Damage:2": 1004,
+              "OreDict:8": "",
+              "id:8": "gregtech:meta_item_1"
             }
           },
-          "taskID:8": "bq_standard:fluid"
+          "taskID:8": "bq_standard:retrieval"
         }
       }
     },
```
Comment (IntegerLimit) [inline overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json:22235]: Why is this quest being replaced?

File: overrides/scripts/_oreDict.zs (hunk 1 above)
Comment (tracer4b) [reply overrides/scripts/_oreDict.zs:2620]: Supposed to be in hm qb

File: overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json (hunk 3 above)
Comment (tracer4b) [reply overrides/config/betterquesting/saved_quests/ExpertQuestsDev.json:22235]: see comments of #367

**Review by IntegerLimit — APPROVED** [review]

**Pull request merged**